### ISSUE TITLE
Error: The provider package isn't installed. Please run `npm install strapi-upload-undefined`

### ISSUE THREAD

**Issue by MirkoVentura** [strapi/strapi#1788]
**Informations**
- **Node.js version*10.8.0*:
- **npm version*6.2.0*:
- **Strapi version*3.0.0-alpha.13.1*:
- **Database*mysql*:
- **Operating system*centos*:

**What is the current behavior?**
After running a created project in production env we had this error 
Error: The provider package isn't installed. Please run `npm install strapi-upload-undefined`
but we want use LOCAL PROVIDER

**Steps to reproduce the problem**
we  follow this guide https://strapi.io/documentation/guides/deployment.html

**What is the expected behavior?**
in our dev machine (mac os or windows) all worked fine

**Suggested solutions**

**Comment by lauriejim** [comment]
@MirkoVentura Can you go in this url of your app `/admin/plugins/upload/configurations/production` and update setting.

**Comment by MirkoVentura** [comment]
Hi @lauriejim I tried now and it does not work i post you a screen of my settings

<img width="1426" alt="upload" src="https://user-images.githubusercontent.com/6473395/44532543-9c0a8900-a6f3-11e8-9929-81caaafb74ba.png">

**Comment by MirkoVentura** [comment]
hear it is for production

<img width="1383" alt="prod" src="https://user-images.githubusercontent.com/6473395/44532616-c9573700-a6f3-11e8-8771-70ca499e7735.png">

**Comment by jrodriguesimg** [comment]
same here using https://github.com/strapi/strapi-docker

**Comment by lauriejim** [comment]
Can you try with fresh new app with last version of Strapi please (3.0.0-alpha.14.1.1)

**Comment by frodoe7** [comment]
I'm facing this same issue
what's the updates please?

**Comment by Thuan2000** [comment]
I'm facing this same issue

Ubuntu server 16.04

**Comment by rijn** [comment]
Seems like a bug in the upload plugin. If you go to the database `core_store` and found the following item
```
17,plugin_upload_provider,"{""enabled"":true,""sizeLimit"":1000}",object,production,""
```
`provider: local` is missing from the schema. Can simply fix by updating the value to
```
{"enabled":true,"provider":"local","sizeLimit":1000}
```

**Comment by kpasargad** [comment]
I'm having the same issue.

**Comment by roghernandez** [comment]
I'm having this issue too, any ideas on what might be causing it? I tried reinstalling Strapi and the upload plugin

**Comment by lauriejim** [comment]
Hello, what is you Strapi version, os, node version and the database you use.

**Comment by Knovour** [comment]
Cannot upload in production mode

Strapi: 3.0.0-alpha.25
OS: CoreOS on Linode
Database: Postgresql

And @rijn 's solution is working

**Comment by webcoderkz** [comment]
@rijn thanks for finding this out, lol. it's very nasty to find this in a production environment, though my dev database doesn't have `"provider": "local"` and uploads work, it's weird.

@lauriejim can you comment on this?

Strapi 3.0.0 - beta 17.4
Node 10, npm 6, local machine Cataina 10.15.1, production Ubuntu 18.04 LTS

**Comment by derrickmehaffy** [comment]
@webcoderkz this is a really old issue? Might be better to open up a new one if you are hitting an issue. This one is from alpha, and a lot of stuff has changed.

**Comment by webcoderkz** [comment]
@derrickmehaffy yes, it's an alpha bug that present on current beta, lol

**Comment by jzyds** [comment]
same error

**Comment by tankadesign** [comment]
The problem still exists running it locally in production mode in beta18.3.

**Comment by jpmayorga** [comment]
Yes i have the same problem specially in production mode on Strapi v3.0.0-beta.18.4

**Comment by alxnkt** [comment]
I'm facing same issue on v3.0.0-beta.18.4. I think, it's bug in save config form https://github.com/strapi/strapi/issues/4510#issuecomment-574869516

**Comment by victorkurauchi** [comment]
I can confirm that @rijn suggestion works as expected! I'm in my staging environment and using aws-s3 sdk

update your entry: `plugin_upload_provider` to have the value
`{"provider":"aws-s3",...`
